### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by IhateTrains**
As usual:
- code cleaning
- customisable localisation fixes
- localisation fixes
- mod and vanilla event fixes
- enabled the owner of Gothic Wars event spawned army - please review this
- other fixes

### CODE REVIEW

File: WTWSMS/localisation/AO_misc.csv [hunk 1]
```diff
@@ -6,3 +6,5 @@ COMPL_WORTHY;worthy;digne;;;;;;;;;;;;x
 INSULT_NAIVE;naive;na�f;;;;;;;;;;;;x
 INSULT_IGNORANT;ignorant;ignorant;;;;;;;;;;;;x
 INSULT_HARSH;harsh;dur;;;;;;;;;;;;x
+GOD_PROMETHEUS;Prometheus;;;;;;;;;;;;;x
+GOD_KRONOS;Kronos;;;;;;;;;;;;;x
```
Comment (loup99) [inline WTWSMS/localisation/AO_misc.csv]: Why did you add this here?

File: WTWSMS/localisation/WtWSMS_misc.csv [hunk 2]
```diff
@@ -224,3 +223,5 @@ rejected_senate;Rejected by Senate;R
 rejected_senate_desc;This character has received the rejected of the Senate.;Ce personnage a �t� r�fut� par le S�nat. ;;;Esta persona ha rechazado al Senadp;;;;;;x
 #### Trade routes;;;;;;;;;;;;;;x
 amber_route;Amber Road;Route de l'ambre;;;;;;;;;;;;x
+#### Misc misc;;;;;;;;;;;;;;x
+String_Requiescat_in_pace;Requiescat in pace;;;;;;;;;;;;;x
```
Comment (loup99) [inline WTWSMS/localisation/WtWSMS_misc.csv]: Wrong file, this isn't added by WtWSMS.

File: WTWSMS/localisation/ACR_cybrxkhan_misc.csv [hunk 3]
```diff
@@ -223,7 +223,6 @@ ancrel.2511.tooltip.2;\n\nSeems like his love spell worked after all. And there
...
 ancrel.2511.tooltip.4;\n\nYou gently tell him he is being silly. �Someday you�ll find a good wife,� you tell him in a sisterly tone. �I can help you find one in the future, if you want.�;\n\nVous dites doucement qu'il est stupide. "Un jour, tu trouveras une bonne femme", dites-vous d'un ton fraternel. "Je peux vous aider � trouver une dans un futur proche, si vous voulez.";;;;;;;;;;x
 ancrel.2511.tooltip.5;\n\nYou gently tell him he is being silly. �Someday you�ll find a good wife,� you tell him in a motherly tone. �I can help you find one in the future, if you want.�;\n\nVous dites doucement qu'il est stupide. "Un jour, tu trouveras une bonne femme", dites-vous d'un ton paternel. "Je peux vous aider � trouver une dans un futur proche, si vous voulez.";;;;;;;;;;x
-ancrel.2511.tooltip.5;\n\nYou gently tell her that he is being silly. �Someday you�ll find a good wife,� you tell her in a motherly tone. �I can help you find one in the future, if you want.�;\n\nVous dites doucement qu'elle est stupide. "Un jour, tu trouveras une bonne femme", dites-vous d'un ton paternel. "Je peux vous aider � trouver une dans un futur proche, si vous voulez.";;;;;;;;;;x
```
Comment (loup99) [inline WTWSMS/localisation/ACR_cybrxkhan_misc.csv]: I'm not sure this should be removed outright, rather than being a 6.

**Comment by loup99** [comment]
- On "fixing" vanilla, I find it outside the scope of the mod, especially given the release of the 2.8 update soon, meaning any eventual fix probably wouldn't be included in any release. Furthermore it is difficult for me as a modder to review content regarding vanilla, as I don't know what the intent behind the script is. Therefore I'm not really able to review that content, but I would put my bets on it being redone in vanilla with the next release. Better to make bug reports than posting it here though, in my opinion. 
- The event troops regarding Narses should in my opinion be disabled because they are far too railroaded and overpowered, without tying into other aspects of the mod. I think those troops being personally owned by Narses don't make much sense either.

File: WTWSMS/localisation/AO_misc.csv (hunk 1 above)
Comment (IhateTrains) [reply WTWSMS/localisation/AO_misc.csv]: these keys were reported missing by the Validator

File: WTWSMS/localisation/ACR_cybrxkhan_misc.csv (hunk 3 above)
Comment (IhateTrains) [reply WTWSMS/localisation/ACR_cybrxkhan_misc.csv]: There's no ancrel.2511.tooltip.6 in the code

File: WTWSMS/localisation/WtWSMS_misc.csv (hunk 2 above)
Comment (IhateTrains) [reply WTWSMS/localisation/WtWSMS_misc.csv]: It's added by me. The string is missing even in vanilla.

Comment (loup99) [reply WTWSMS/localisation/WtWSMS_misc.csv]: Then I would add it in WtWSMS_override.csv.

File: WTWSMS/localisation/ACR_cybrxkhan_misc.csv (hunk 3 above)
Comment (loup99) [reply WTWSMS/localisation/ACR_cybrxkhan_misc.csv]: In ACR it is ancrel.2511.tooltip.6. I would advice that we stick to that.

File: WTWSMS/localisation/AO_misc.csv (hunk 1 above)
Comment (loup99) [reply WTWSMS/localisation/AO_misc.csv]: Okay.

**Comment by loup99** [comment]
- Report them by posting the fixed code. You don't need to do much more than that.
- Disabled, in my opinion.

**Comment by IhateTrains** [comment]
* If that's so simple, I could do that, but I'm afraid these files have been either slightly adapted for WtWSMS or not completely merged with the latest patches

File: WTWSMS/localisation/WtWSMS_misc.csv (hunk 2 above)
Comment (IhateTrains) [reply WTWSMS/localisation/WtWSMS_misc.csv]: ok, moved

File: WTWSMS/localisation/ACR_cybrxkhan_misc.csv (hunk 3 above)
Comment (IhateTrains) [reply WTWSMS/localisation/ACR_cybrxkhan_misc.csv]: ok, done

File: WTWSMS/localisation/WtWSMS_misc.csv (hunk 2 above)
Comment (IhateTrains) [reply WTWSMS/localisation/WtWSMS_misc.csv]: ok, moved

File: WTWSMS/localisation/ACR_cybrxkhan_misc.csv (hunk 3 above)
Comment (IhateTrains) [reply WTWSMS/localisation/ACR_cybrxkhan_misc.csv]: ok, done

**Comment by IhateTrains** [comment]
Are all things settled?

File: WTWSMS/events/job_lord_spiritual.txt [hunk 4]
```diff
@@ -2757,7 +2757,7 @@ character_event = {
 							text = WAGING_CRUSADE
 							hidden_tooltip = {
 								using_cb = crusade
-								using_cb = religious
+								#using_cb = religious
```
Comment (loup99) [inline WTWSMS/events/job_lord_spiritual.txt]: Why did you disable this and the below? Isn't it just missing an "OR"?

File: WTWSMS/localisation/WtWSMS_cultures.csv [hunk 5]
```diff
@@ -818,7 +817,7 @@ damnonii;Damnonii;damnonii;;;Dumnonia;;;;;;;x
 venedi;Venedi;wende;;;V�neda;;;;;;;x
 cantabrian;Cantabrian;cantabre;;;C�ntabra;;;;;;;x
 khwarezmian;Khwarezmian;chorasmien;;;Khwarezmia;;;;;;;x
-latin;Latin;latin;;;Latina;;;;;;;x
+#latin;Latin;latin;;;Latina;;;;;;;x
```
Comment (loup99) [inline WTWSMS/localisation/WtWSMS_cultures.csv]: Why did you disable this?

Comment (IhateTrains) [reply WTWSMS/localisation/WtWSMS_cultures.csv]: duplicate with vanilla

File: WTWSMS/events/job_lord_spiritual.txt (hunk 4 above)
Comment (IhateTrains) [reply WTWSMS/events/job_lord_spiritual.txt]: two instances of using_cb (look at line 2759)

Comment (IhateTrains) [reply WTWSMS/events/job_lord_spiritual.txt]: I can add "OR" and re-enable the second CB.

Comment (IhateTrains) [reply WTWSMS/events/job_lord_spiritual.txt]: Done

File: WTWSMS/localisation/WtWSMS_cultures.csv (hunk 5 above)
Comment (loup99) [reply WTWSMS/localisation/WtWSMS_cultures.csv]: The localisation isn't for the group though, it is for the culture.

Comment (loup99) [reply WTWSMS/localisation/WtWSMS_cultures.csv]: It is intended for the Latin culture added by the mod though, not the Latin culture group from vanilla.

File: WTWSMS/events/job_lord_spiritual.txt (hunk 4 above)
Comment (loup99) [reply WTWSMS/events/job_lord_spiritual.txt]: Yes, because OR is definitely what is intended.

File: WTWSMS/localisation/WtWSMS_cultures.csv (hunk 5 above)
Comment (IhateTrains) [reply WTWSMS/localisation/WtWSMS_cultures.csv]: Latin being a culture group is not a problem here. The looked at the localisation in vanilla again and the mod one is better. I will enable it.

Comment (IhateTrains) [reply WTWSMS/localisation/WtWSMS_cultures.csv]: done
